.github/containerscan/allowedlist.yaml

```diff
@@ -12,6 +12,7 @@ general:
     - CVE-2022-0778
     - CVE-2022-2526
     - CVE-2022-27664
```

Comment:
Should CVE-2022-27664 be removed implementing the workaround found by the Yulan team?

Reply:
Yes when we implement that